commit 85fbb4148f349e8919ad3e308fa4448866d53899 (refs/changes/16/516/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-09-21T19:18:47+00:00 (3 years, 1 month ago)
    
    IMGAPI-571 re-do images.jo/updates.jo deployment to be able use stock 'imgapi' images 
    IMGAPI-567 Update stud for OpenSSL 1.0.1t/1.0.2h vulnerability fixes

diff --git a/build.spec b/build.spec
index 157abea6..d44f98ef 100644
--- a/build.spec
+++ b/build.spec
@@ -136,6 +136,12 @@
       "name": "sdc-base",
       "version": "14.2.0",
       "uuid": "de411e86-548d-11e4-a4b7-3bb60478632a"
-    }
+    },
+    "sdc-minimal-multiarch-lts-15.4.1": {
+      "imgapi": "https://updates.joyent.com",
+      "name": "sdc-minimal-multiarch-lts",
+      "version": "15.4.1",
+      "uuid": "18b094b0-eb01-11e5-80c1-175dac7ddf02"
+    } 
   }
 }
diff --git a/config/sapi/services/imgapi/service.json b/config/sapi/services/imgapi/service.json
index 3a202415..62d0d344 100644
--- a/config/sapi/services/imgapi/service.json
+++ b/config/sapi/services/imgapi/service.json
@@ -15,7 +15,6 @@
     },
     "metadata": {
         "SERVICE_NAME": "imgapi",
-        "STORAGE_LOCAL_BASEDIR": "/data/imgapi",
         "STORAGE_MANTA_BASEDIR": "imgapi"
     },
     "manifests": {
